packages/graphql/tests/integration/indexes-and-constraints/fulltext.int.test.ts

```diff
@@ -530,4 +530,132 @@ describe("assertIndexesAndConstraints/fulltext", () => {
             `@fulltext index '${indexName}' on Node '${type.name}' is missing field 'description' aliased to field '${alias}'`
         );
     });
+
+    test("should create index if it doesn't exist and not throw if it does exist, and then query using the index", async () => {
+        // Skip if multi-db not supported
+        if (!MULTIDB_SUPPORT) {
```

Comment:
**thought (non-blocking)** I see this is repeated maybe we can make a function to abstract this.